conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, CMake, tools
+
+
+class HelloConan(ConanFile):
+    name = "LibConfig"
```

Comment:
Conan 2.0 might not accept upper cases in recipe names.
I guess for support with Windows.
```suggestion
    name = "libconfig"
```

Reply:
thanks, didn't know that.